Describe aws/gcp actions to configure IAM roles/Workload ID for the O… (#353)
okteto/docs: versioned_docs/version-1.6/self-hosted/administration/configuration.mdx

### publicOverride

> Use this property to override the Public URL where Okteto is available. This option replaces `okteto.$SUBDOMAIN` with your [FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) of choice.

Reviewer: ```suggestion
Use this property to override the Public URL where Okteto is available. This option replaces `okteto.$SUBDOMAIN` with the [FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) you provided.
```
